plugins/woocommerce/src/StoreApi/Routes/V1/Agentic/Messages/Messages.php

```diff
@@ -0,0 +1,56 @@
+<?php
+declare(strict_types=1);
+
+namespace Automattic\WooCommerce\StoreApi\Routes\V1\Agentic\Messages;
+
+/**
+ * Class Messages
+ *
+ * Manages error & info messages for the agentic checkout process.
+ */
+class Messages {
```

Comment:
Nitpick: to better distinguish with `abstract Message`.
```suggestion
class MessageCollection {
```

Reply:
I'll address this on the next iteration of error behavior :)